python/paddle/fluid/tests/unittests/test_imperative_checkpoint.py

```diff
@@ -102,29 +102,38 @@ class TestDygraphCheckpoint(unittest.TestCase):
     def test_save_load_persistables(self):
         seed = 90
         epoch_num = 1
+        batch_size = 128
 
         with fluid.dygraph.guard():
             fluid.default_startup_program().random_seed = seed
             fluid.default_main_program().random_seed = seed
 
             mnist = MNIST("mnist")
             sgd = SGDOptimizer(learning_rate=1e-3)
-            train_reader = paddle.batch(
-                paddle.dataset.mnist.train(), batch_size=128, drop_last=True)
+
```

Comment:
Please Add another UT to test reader instead of using test_imperative_checkpoint.py

Reply:
great